charts/seed-bootstrap/charts/etcd-druid/Chart.yaml

```
@@ -0,0 +1,4 @@
+apiVersion: v1alpha1
```

```suggestion
apiVersion: v1
```
```
apiVersion: The chart API version, always "v1" (required)
```
ref: https://v2.helm.sh/docs/developing_charts/#charts